### PR TITLE
support allowPatialResult

### PR DESCRIPTION
**Pull request by stuBirdFly**
## Summary

support allowPatialResult

### CODE REVIEW

File: src/main/java/com/alipay/oceanbase/hbase/OHTable.java [hunk 1]
```diff
@@ -478,8 +508,16 @@ public Result call() throws IOException {
                         || get.getFamilyMap().keySet().size() == 0) {
                         filter = buildObHTableFilter(get.getFilter(), get.getTimeRange(),
                             get.getMaxVersions(), null);
-                        obTableQuery = buildObTableQuery(filter, get.getRow(), true, get.getRow(),
-                            true);
+                        if(get.isClosestRowBefore()){
+                            obTableQuery = buildObTableQuery(filter, null, false, get.getRow(),
+                                    true);
+                            obTableQuery.setBatchSize(1);
```
Comment (shenyunlong) [inline src/main/java/com/alipay/oceanbase/hbase/OHTable.java:514]: setCaching?

File: src/main/java/com/alipay/oceanbase/hbase/OHTable.java [hunk 2]
```diff
@@ -493,8 +531,16 @@ public Result call() throws IOException {
                             filter = buildObHTableFilter(get.getFilter(), get.getTimeRange(),
                                 get.getMaxVersions(), entry.getValue());
 
-                            obTableQuery = buildObTableQuery(filter, get.getRow(), true,
-                                get.getRow(), true);
+                            if(get.isClosestRowBefore()){
+                                obTableQuery = buildObTableQuery(filter, null, false, get.getRow(),
+                                        true);
+                                obTableQuery.setBatchSize(1);
```
Comment (shenyunlong) [inline src/main/java/com/alipay/oceanbase/hbase/OHTable.java:537]: setCaching?

File: pom.xml [hunk 3]
```diff
@@ -58,6 +58,13 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>obkv-table-client</artifactId>
+            <version>1.2.100-SNAPSHOT</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/obkv-table-client-1.2.100-SNAPSHOT-jar-with-dependencies.jar</systemPath>
```
Comment (shenyunlong) [inline pom.xml:66]: is this a tmp table-client jar?

File: src/main/java/com/alipay/oceanbase/hbase/OHTable.java (hunk 1 above)
Comment (stuBirdFly) [reply src/main/java/com/alipay/oceanbase/hbase/OHTable.java]: fixed

File: pom.xml (hunk 3 above)
Comment (stuBirdFly) [reply pom.xml]: fixed

File: src/main/java/com/alipay/oceanbase/hbase/OHTable.java (hunk 2 above)
Comment (stuBirdFly) [reply src/main/java/com/alipay/oceanbase/hbase/OHTable.java]: fixed

File: pom.xml (hunk 3 above)
Comment (stuBirdFly) [reply pom.xml]: This is the `obkv_params` branch of the repository at <https://github.com/oceanbase/obkv-table-client-java/tree/obkv_params>.

**Comment by shenyunlong** [comment]
LGTM

**Pull request merged**